src/DependencyInjection/SurfnetSamlExtension.php

```diff
@@ -158,9 +158,14 @@ private function parseRemoteIdentityProviderConfiguration(array $identityProvide
         $configuration = [
             'entityId' => $identityProvider['entity_id'],
             'ssoUrl' => $identityProvider['sso_url'],
-            'certificateData' => $identityProvider['certificate'],
         ];
 
+        if ( isset($identityProvider['certificate_file']) ) {
+            $configuration['certificateFile'] = $identityProvider['certificate_file'];
+        } else {
+            $configuration['certificateData'] = $identityProvider['certificate'];
+        }
```

Comment:
I'd rather verify one of these is set, so that specifying neither results in an error, and specifying both. This way there can be no mistake which certificate is used.

On another note: @thijskh, could you enable Travis CI for the SURFnet/Stepup-saml-bundle repository?